### PR TITLE
PS-7641 add procFS pluging

### PR DESCRIPTION
**Pull request by vorsel**

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by george-lorch** [comment]
Please do not merge this until https://github.com/percona/percona-server/pull/4251 has been merged.  Thanks!

**Comment by vorsel** [comment]
> Please do not merge this until #4251 has been merged. Thanks!

Sure, noted

**Comment by george-lorch** [comment]
@vorsel https://github.com/percona/percona-server/pull/4251 has been merged for 8.0.25 release, please merge this when ready.  Thanks!

**Pull request merged**
